codebase2/codebase-sqlite/U/Codebase/Sqlite/Queries.hs

```diff
@@ -1291,11 +1309,17 @@ getDependentsForDependency dependency =
           AND dependency_component_index IS ?
       |]
 
+    isNotReferenceFromOwnComponent :: Reference.Id -> Bool
+    isNotReferenceFromOwnComponent =
```

Comment:
is `isNotSelfComponent` any clearer? Maybe 1291 could also be `ExcludeSelfComponent`